packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/Distance.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.kotlin.types
```

Comment:
I was thinking the same thing. .NET has done that and we have also done that with the dynamic API, maybe I should move all of these classes to `io.realm.kotlin.geo`? And move them out of `types` since that is only our persistable types.

Reply:
I would vote for a separate package ... but some of them will eventually be persistable types, or what?